### PR TITLE
Tempura norm

### PR DESCRIPTION
**Pull request by qujia7**
changed import from cmblensplus to pytempura.
Small change in the initial arguments in simple.py and removed args.curl redundant in solenspipe.py.

### CODE REVIEW

File: solenspipe/solenspipe.py [hunk 1]
```diff
@@ -740,7 +738,7 @@ def cmblensplus_norm(nltt,nlee,nlbb,theory,theory_cross,lmin,lmax):
 
 def diagonal_RDN0(get_sim_power,nltt,nlee,nlbb,theory,theory_cross,lmin,lmax,simn):
     """Curvedsky dumb N0 for TT,EE,EB,TE,TB"""
-    import curvedsky as cs
+    import tempura as cs
```
Comment (msyriac) [inline solenspipe/solenspipe.py]: Shouldn't this be pytempura?

File: solenspipe/solenspipe.py [hunk 2]
```diff
@@ -802,7 +800,7 @@ def diagonal_RDN0(get_sim_power,nltt,nlee,nlbb,theory,theory_cross,lmin,lmax,sim
     
 def diagonal_RDN0mv(get_sim_power,nltt,nlee,nlbb,theory,theory_cross,lmin,lmax,simn):
     """Curvedsky dumb N0 for MV"""
-    import curvedsky as cs
+    import tempura as cs
```
Comment (msyriac) [inline solenspipe/solenspipe.py]: Similarly here and in several other places.

**Review by msyriac — CHANGES_REQUESTED** [review]

File: solenspipe/solenspipe.py (hunk 1 above)
Comment (NiallMac) [reply solenspipe/solenspipe.py]: (Which is also imported at the top of the file, so does not need to be imported again anyway)

Comment (qujia7) [reply solenspipe/solenspipe.py]: yes, I added previously the import on the top of the file but forgot to remove some of the previous imports. Those are removed now.

**Comment by msyriac** [comment]
Thanks, merging soon. I'm guessing @NiallMac and @qujia7 have been able to install tempura successfully? It isn't a PyPI package yet, but should be able to do that soon.

**Review by msyriac — APPROVED** [review]

**Pull request merged**